controllers/clusters/cadence_controller.go

```diff
@@ -60,10 +60,6 @@ type CadenceReconciler struct {
 
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
-// TODO(user): Modify the Reconcile function to compare the state specified by
-// the Cadence object against the actual cluster state, and then
-// perform operations to make the cluster state reflect the state specified by
-// the user.
```

Comment:
Why did you remove this comment?

Reply:
I think It's unneeded part of comment, we have removed it in other controllers